erb_lint.gemspec

```diff
@@ -11,7 +11,7 @@ Gem::Specification.new do |s|
   s.email = ['justin.the.c@gmail.com']
   s.summary = 'ERB lint tool'
   s.description = 'ERB Linter tool.'
-  s.homepage = 'https://github.com/justinthec/erb-lint'
+  s.homepage = 'https://github.com/Shopify/erb-lint'
```

Comment:
If the maintainers wish to continue honoring the creator by pointing to the original repo, then this could be modified to [use the metadata hash to specify a source code URL](https://guides.rubygems.org/specification-reference/#metadata), e.g.:
```
s.metadata = { source_code_uri: 'https://github.com/Shopify/erb-lint' }
```